File: LATIHAN2/Human.java
// class untuk NIK, NAMA, JK
public class Human { // public agar bebas diakses
  private String NIK;
  private String nama;
  private String jk;

  // Konstruktor
  public Human(String NIK, String nama, String jk) {
    this.NIK = NIK;
    this.nama = nama;
    this.jk = jk;
  }

  // Print NIK, NAMA, JK
  public void display() {
    System.out.println("NIK           : " + NIK);
    System.out.println("Nama          : " + nama);
    System.out.println("Jenis kelamin : " + jk);
  }

  // getter setter
  public String getNIK() { return NIK; }
  public void setNIK(String NIK) { this.NIK = NIK; }

  public String getNama() { return nama; }
  public void setNama(String nama) { this.nama = nama; }

  public String getJk() { return jk; }
  public void setJk(String jk) { this.jk = jk; }
}

File: LATIHAN2/PHP/Human.php
<?php
//kelas human
class Human {
  private $NIK;
  private $nama;
  private $jk;

    //constructor
  public function __construct($NIK, $nama, $jk) {
    $this->NIK = $NIK;
    $this->nama = $nama;
    $this->jk = $jk;
  }
  //methode menampilkan isi variabel
  public function display() {
    echo "NIK           : " . $this->NIK . "<br>";
    echo "Nama          : " . $this->nama . "<br>";
    echo "Jenis kelamin : " . $this->jk . "<br>";
  }

  //ggetter setter
  public function getNIK() { return $this->NIK; }
  public function setNIK($NIK) { $this->NIK = $NIK; }

  public function getNama() { return $this->nama; }
  public function setNama($nama) { $this->nama = $nama; }

  public function getJk() { return $this->jk; }
  public function setJk($jk) { $this->jk = $jk; }
}
